### PR TITLE
fix: multi-refreshes without purpose, critical code smells, inconsistent imports and legacy ESL usages

### PR DESCRIPTION
**Pull request by ala-n**
Closes: #526

In the scope of the current PR:
- Fix multiple refreshes issue #526 ; Now refresh happens just once
- Simplify and remove useless plugins and wrappers for `uip-editor`
- Create a stable and predictable normalization process on the core level to use an architecturally correct normalized comparative process
- Remove usages of legacy ESL aliases
- Reorder and simplify imports (lint rule will be added in the separate request)
- Move utils to the proper directories
- Remove MSG object - move to the settings module constants instead
- Partially remove less that is not related to project output
- Fix less output of the project

### CODE REVIEW

File: src/core/base/plugin.less [hunk 1]
```diff
@@ -1,3 +1,6 @@
+@import (reference) "../../common/mixins.less";
+@import (reference) "../../common/variables.less";
```
Comment (NastaLeo) [inline src/core/base/plugin.less:2]: unnecessary import

File: src/plugins/settings/setting.less [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import (reference) "../../common/variables.less";
```
Comment (NastaLeo) [inline src/plugins/settings/setting.less:1]: unnecessary import

File: src/settings/bool-setting/bool-setting.less [hunk 3]
```diff
@@ -1,3 +1,5 @@
+@import (reference) "../../common/variables.less";
+
```
Comment (NastaLeo) [inline src/settings/bool-setting/bool-setting.less:2]: unnecessary import

File: src/settings/slider-setting/slider-setting.less [hunk 4]
```diff
@@ -1,3 +1,5 @@
+@import (reference) "../../common/variables.less";
```
Comment (NastaLeo) [inline src/settings/slider-setting/slider-setting.less:1]: unnecessary import

**Review by NastaLeo — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Review by AlexanderBazukevich — APPROVED** [review]

**Pull request merged**